bunsen/bunsen-avro/src/test/java/com/cerner/bunsen/avro/R4AvroConverterTest.java

```diff
@@ -285,8 +282,8 @@ public void testMultiReferenceTypes() {
 
     Record practitioner = (Record) ((List) avroPatient.get("generalPractitioner")).get(0);
```

Comment:
Do we have a unit-test that confirms the `id` elements are dropped from nested fields? If not, can you please add one?

Reply:
added